docs/2.1. APIs - Client Side Implementation.md

```diff
@@ -4,19 +4,16 @@ _(c) AMWA 2017, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
 
 ## Conforming to Schemas
 
-When interacting with a Connection Management API, a client may assume the presence of a core set of attributes for each defined transport type. Other parameters may be defined by the specification, but are optional for implementations given that they may or may not implement a particular feature. Presence of these attributes in an implementation cannot be assumed and must be identified via the '/constraints' resource of a running Device. A list of core attributes, and attributes associated with particular features, is given in [Behaviour - RTP Transport Type](4.1.%20Behaviour%20-%20RTP%20Transport%20Type.md) and further pages in the [Behaviour](4.0.%20Behaviour.md) section.
-
-For core attributes, individual APIs may still constrain their permitted values to a particular range or enumeration, so parsing the constraints is still recommended.
-
+The Connection Management API schemas define sets of parameters for each supported transport type; these are also described in pages in the [Behaviour](4.0.%20Behaviour.md) section of these documents. The schemas define which parameters are always present (core), and which are feature-dependent; a client can determine what a Device supports by getting its `/constraints` resource, and SHOULD parse this resource in all cases as a Device might have a constrained range or enumeration of supported values, even for core paramters.
```

Comment:
There's a typo in the last word of the paragraph.
But more importantly, I'm not sure this is true as now written. The schemas _do_ define the parameters for each supported transport type. But they _don't_ define the _sets of parameters_ nor which ones are core. That, unfortunately, is _only_ done by the documentation in the relevant subsections.